tests/countries/test_turks_and_caicos_islands.py

```
@@ -0,0 +1,156 @@
+                dt = date(2020, 5, 25)
+            elif year == 2021:
+                dt = date(2021, 5, 31)
+            elif year == 2022:
+                dt = date(2022, 5, 30)
+            elif year == 2023:
+                dt = date(2023, 5, 29)
+            elif year == 2024:
+                dt = date(2024, 5, 27)
+            else:  # 2025
+                dt = date(2025, 5, 26)
+                
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)